django/http/response.py

```diff
@@ -609,9 +609,11 @@ def set_headers(self, filelike):
                 # Encoding isn't set to prevent browsers from automatically
                 # uncompressing files.
                 content_type = {
+                    "br": "application/x-br",
                     "bzip2": "application/x-bzip",
                     "gzip": "application/gzip",
                     "xz": "application/x-xz",
+                    "compress": "application/x-compress",
```

Comment:
Not sure if these are acceptable mime types :thinking:

Reply:
[application/x-compress](https://www.digipres.org/formats/sources/ffw/formats/#compress%20(unix)) looks ok to me :+1: 

I couldn't find a mime type for [brotli](https://www.digipres.org/formats/sources/ffw/formats/#brotli). I can find discussions about it :thinking: 

But I'm not confident where I am looking is the best place

Reply:
Thanks for checking :+1: Some related issues/reports suggest `x-brotli` so it should work for now 

- https://bugs.astron.com/view.php?id=111
- https://github.com/google/brotli/issues/727
- https://github.com/google/brotli/issues/299